acai-ios/Acai.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  Acai.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/5/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class Acai {
+
+    static let bowlOrderBaseOptionsIdentifier: String = "bowlOrderBaseOptionsIdentifier"
+    static let bowlOrderToppingOptionsIdentifier: String = "bowlOrderToppingOptionsIdentifier"
+    static let detailCollectionHeaderIdentifier: String = "detailCollectionHeaderIdentifier"
+    
+    static let avenirNextMedium: UIFont = UIFont(name: "AvenirNext-Medium", size: 12)!
+    static let avenirNextDemi: UIFont = UIFont(name: "AvenirNext-DemiBold", size: 12)!
+    static let avenirNextBold: UIFont = UIFont(name: "AvenirNext-Bold", size: 12)!
```

Comment:
you should make these in a `UIFont+Shared.swift` and add these as an extension to `UIFont`